### PR TITLE
Clesaec/tcomp 1733 log4j pb

### PR DESCRIPTION
**Pull request by clesaec**
**What is the current behavior?** (You can also link to an open issue here)

https://jira.talendforge.org/browse/TCOMP-1733

**What is the new behavior?**

**Please check if the PR fulfills these requirements**

- [ ] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.sdk.component.studio-integration/pom.xml [hunk 1]
```diff
@@ -109,17 +109,17 @@
       <artifactId>johnzon-jsonb</artifactId>
     </dependency>
 
-    <dependency>
+    <!--<dependency>
```
Comment (ypiel-talend) [inline main/plugins/org.talend.sdk.component.studio-integration/pom.xml:112]: Remove the dependency if not used.

File: main/plugins/org.talend.sdk.component.studio-integration/pom.xml [hunk 2]
```diff
@@ -163,18 +163,18 @@
       <artifactId>slf4j-api</artifactId>
       <version>${slf4j.version}</version>
     </dependency>
-    <dependency>
+    <!--<dependency>
```
Comment (ypiel-talend) [inline main/plugins/org.talend.sdk.component.studio-integration/pom.xml:166]: REmove dependency if not used.

File: main/plugins/org.talend.sdk.component.studio-integration/pom.xml [hunk 3]
```diff
@@ -163,18 +163,18 @@
...
-    <dependency>
+    <!--<dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-simple</artifactId>
       <version>${slf4j.version}</version>
-    </dependency>
+    </dependency>-->
     <!-- Replaces log4j dep which is used in Studio plugin -->
-    <dependency>
+  <!--  <dependency>
```
Comment (ypiel-talend) [inline main/plugins/org.talend.sdk.component.studio-integration/pom.xml:172]: Same...

File: main/plugins/org.talend.sdk.component.studio-integration/pom.xml (hunk 1 above)
Comment (clesaec) [reply main/plugins/org.talend.sdk.component.studio-integration/pom.xml:112]: It's a temporary remove, until we find a compliant solution to comile and pass unit tests

**Pull request merged**